README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are required to comply with the following code of conduct. HangOps admins are responsible for ensuring that all community members follow this code within HangOps spaces.
+
+## The Short Version
+
+Be respectful of other people; respectfully ask people to stop if you are bothered; and if you can’t resolve an issue contact the admins. If you do not follow our Community Code of Conduct, it will be apparent and you may be removed from HangOps.
```

Comment:
Consider the following "apology":

> I'm sorry if you felt disrespected.

It's mealy-mouthed and targets the victim instead of the perpetrator. So does this, for much the same reason:

> respectfully ask people to stop if you feel disrespected

Maybe try something like:

> The community should respectfully call out anybody behaving inappropriately, ask them to stop, and direct them to this code of conduct.

That way it's obvious that the problem lies with the perpetrator, not the victim.